api_tests/logs/views/test_log_detail.py

```diff
@@ -1,142 +1,200 @@
-# -*- coding: utf-8 -*-
```

Comment:
I'm not 100% sure about this, but I think that they want that comment to stay there? @alexschiller 

I might have missed this in your other PR's